{"name": "test_add2[【类型】输入为空，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：-3和开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660805207768, "stop": 1660805207768}], "attachments": [{"name": "log", "source": "20a3fc40-02a2-48d1-9b63-47fee58fb323-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "9e05299e-fd2d-4db4-bff3-285a71c6c1db-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "cbfc872c-99cd-4ddb-a3ab-64e9665476cb-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "-3"}, {"name": "y", "value": "''"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660805207767, "stop": 1660805207768, "uuid": "8b6b37fb-c247-44e9-a1ad-bfa1cc216c36", "historyId": "0412f9eaf97e435531107a7fcdc7ef69", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "7552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}